pontoon/settings/base.py

```diff
@@ -1127,14 +1130,23 @@ def account_username(user):
 except ValueError as e:
     raise ValueError(f"Error: {e}")
 
-# Used for Review Master and Translation Champion
+# Used for Translation Champion badge
 BADGES_TRANSLATION_THRESHOLDS = list(
     map(
         int,
         os.environ.get("BADGES_TRANSLATION_THRESHOLDS", "5, 50, 250, 1000").split(","),
     )
 )
-# Used for Community Builder
+# Used for Review Master badge
+# Thresholds are higher than Translation Champion to minimize
+# the chances that both badges are awarded at the same exact time.
```

Comment:
```suggestion
# Thresholds are higher than Translation Champion because implicit
# rejections on translations submissions and approvals also count against the Review Master badge
```